### DISCUSSION TITLE
Passaggio al nuovo anno 2024/2025

### DISCUSSION THREAD

**Discussion by luigirotondi2009** [iisgiua/giuaschool#710] [Richieste di aiuto]
Salve, quando è consigliabile il cambio anno? Come posso fare il passaggio di anno correttamente senza che danneggi database e tutti i dati? Usiamo Altervista.

**Comment by trinko** [comment]
La versione attuale non è testata per il passaggio al nuovo anno, per cui sconsiglio di farlo adesso.
Saranno rilasciati aggiornamenti in seguito e scriverò negli annunci quando uscirà la versione testata per l'archiviazione e il passaggio al nuovo anno.

Attualmente prevedo che sarà tutto completato entro luglio.
